### PR TITLE
Improve glossary

### PR DESCRIPTION
**Pull request by cbrnr**
I read through the glossary and made some improvements (I hope).

Open questions:

1. I would capitalize the terms for aesthetic reasons.
2. We could add the Glossary to the left sidebar on our homepage. Thoughts?

### CODE REVIEW

**Review by agramfort — APPROVED** [review]

File: doc/glossary.rst [hunk 1]
```diff
@@ -381,17 +377,11 @@ general neuroimaging concepts. If you think a term is missing, please consider
...
+        structure into "whitened data", which has a covariance structure equal to
+        the identity matrix. In other words, whitening creates virtual channels that
         are uncorrelated and have unit variance. This is also known as a
         sphering transformation.
 
-        The term "whitening" comes from the fact that light with a flat
-        frequency spectrum in the visible range is white, whereas
-        non-uniform frequency spectra lead to perception of different colors
-        (e.g., "pink noise" has a ``1/f`` characteristic, which for visible
-        light would appear pink).
```
Comment (larsoner) [inline doc/glossary.rst:393]: This was added intentionally, please put it back (see e.g. the discussion in https://github.com/scipy/scipy/issues/12446#issuecomment-683454152 / https://github.com/mne-tools/mne-python/issues/7425#issuecomment-679324485)

**Review by larsoner — COMMENTED** [review]
> I would capitalize the terms for aesthetic reasons.

-1 on this, I think it's fine as is, and is consistent with sklearn: https://scikit-learn.org/stable/glossary.html

> We could add the Glossary to the left sidebar on our homepage. Thoughts?

To me having it prominent in the left sidebar of the Documentation section as it is currently is the correct scope:

https://mne.tools/dev/overview/index.html

File: doc/glossary.rst (hunk 1 above)
Comment (cbrnr) [reply doc/glossary.rst:393]: Ah, thanks for the links, I'd never have thought that this could be offensive. Sure, I'll put it back.

**Comment by cbrnr** [comment]
This is ready to be merged.

**Comment by larsoner** [comment]
Thanks @cbrnr !

**Pull request merged**